src/app/files.js

```diff
@@ -138,12 +138,7 @@ function Files (storage) {
     var tree = {}
 
     var self = this
-    storage.keys().forEach(function (path) {
-      // NOTE: as a temporary measure do not show the config file
-      if (path === '.remix.config') {
-        return
-      }
-
+    Object.keys(this.list()).forEach(function (path) {
```

Comment:
I guess adding a comment here could be useful to say that since it uses `list()`, the `.remix.config` is removed form the list.